internal/storage/chunk_uploader_test.go

```diff
@@ -0,0 +1,588 @@
+package storage
```

Comment:
there are a lot of common assertions in this test file, can we move them to a common helper function?

Reply:
Clubbed all the common assertions around the new/upload/close calls into helper functions. 

I am not sure it's good idea to try to optimize/share the test logic easily without making it harder to change them later if needed.

Reply:
@ashmeenkaur reminder.

Reply:
Thank you!